src/Components/WriteOnlyButton/WriteOnlyButton.js

```diff
@@ -3,33 +3,44 @@ import React from 'react';
 import { useQueryClient } from 'react-query';
 import PropTypes from 'prop-types';
 import NoAccessPopover from '../NoAccessPopover';
+import { Tooltip } from '@patternfly/react-core';
 
 const WriteOnlyButton = (props) => {
-  const { children, ...buttonProps } = props;
+  const { children, enabledTooltip, disabledTooltip, ...buttonProps } = props;
```

Comment:
I would make disabledTooltip have a default value because this will make all other ones break and gate the enabledTooltip around if its defined - again because this will make every WriteOnlyButton expect these props

Reply:
> and gate the enabledTooltip around if its defined

Right now every writeOnlyButton that doesn't have a tooltip normally looks like this
![image](https://github.com/RedHatInsights/sed-frontend/assets/29445377/a0c57391-2bd5-4b6d-b463-fa480b5401b3)